### DISCUSSION TITLE
Our translation is waiting for validation

### DISCUSSION THREAD

**Discussion by NathalieTalmasse** [SiimKirjanen/seatreg#144] [Q&A]
Good morning,
A few weeks ago Autre Terre made a 5 words translation into French to see how it goes.  The translation is qtill waiting for validation.  
When will you validate the translation ? 
Once we will translate all the termes we need beginning of August, how long will it take for them to be validated ?  
Thanks for your answer,
Nathalie Talmasse
nathalie.talmasse@autreterre.org

**Comment by SiimKirjanen** [comment]
Hello
We need someone who has permissions to approve French translations. Sadly I don't have those permissions.
I think there is a way to request the translation review or even become a translation editor.
Example: https://make.wordpress.org/polyglots/2021/10/03/seatreg-translated-to-danish-howto/ and https://make.wordpress.org/polyglots/2023/03/25/pte-request-for-seatreg-2/

**Comment by SiimKirjanen** [comment]
French translation team: https://make.wordpress.org/polyglots/teams/?locale=fr_FR

**Comment by NathalieTalmasse** [comment]
Hello,

How can we become a French translation editor ?

Thanks for the explanations,
Nathalie


Nathalie Talmasse





Autre Terre asbl<www.autreterre.org>


Tél:+32 4 240 68 40


N° Int:6840



Fax:+32 4 228 07 50

De : Siim Kirjanen ***@***.***
Envoyé : mardi 27 juin 2023 14:40
À : SiimKirjanen/seatreg ***@***.***>
Cc : Nathalie Talmasse ***@***.***>; Author ***@***.***>
Objet : Re: [SiimKirjanen/seatreg] Our translation is waiting for validation (Discussion #144)


Hello
We need someone who has permissions to approve French translations. Sadly I don't have those permissions.
I think there is a way to request the translation review or even become a translation editor.
Example: https://make.wordpress.org/polyglots/2021/10/03/seatreg-translated-to-danish-howto/ and https://make.wordpress.org/polyglots/2023/03/25/pte-request-for-seatreg-2/

—
Reply to this email directly, view it on GitHub<https://github.com/SiimKirjanen/seatreg/discussions/144#discussioncomment-6293446>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BA37AE4DBKC3MXZY2RHCYG3XNLIARANCNFSM6AAAAAAZVKVOKM>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

 [https://www.groupeterre.org/sites/groupeterre/files/Logo_pour_signature/logo-mail-AT.png] <https://www.autreterre.org>


Nathalie Talmasse


Assistante de direction



Tél:+32 4 240 68 40


Fax:+32 4 228 07 50


***@***.******@***.***>


Rejoignez Autre Terre sur Facebook<http://www.facebook.com/ongautreterre>



Autre Terre asbl


4e Avenue, 45


B-4040 Herstal


Dossiers et Comptabilité




[https://www.groupeterre.org/sites/groupeterre/files/Logo_pour_signature/iconNewsAT.png]Inscrivez-vous à la newsletter d'Autre terre<https://my.sendinblue.com/users/subscribeembed/js_id/2ml2i/id/1>

**Comment by SiimKirjanen** [comment]
Hello
You can submit a request using https://make.wordpress.org/polyglots/ form.

**Comment by NathalieTalmasse** [comment]
Hello,

I’m sorry but can not find the form.
Could you help me a bit more ?
Thank you,
Nathalie



Nathalie Talmasse





Autre Terre asbl<www.autreterre.org>


Tél:+32 4 240 68 40


N° Int:6840



Fax:+32 4 228 07 50

De : Siim Kirjanen ***@***.***
Envoyé : lundi 17 juillet 2023 17:41
À : SiimKirjanen/seatreg ***@***.***>
Cc : Nathalie Talmasse ***@***.***>; Author ***@***.***>
Objet : Re: [SiimKirjanen/seatreg] Our translation is waiting for validation (Discussion #144)


Hello
You can submit a request using https://make.wordpress.org/polyglots/ form.

—
Reply to this email directly, view it on GitHub<https://github.com/SiimKirjanen/seatreg/discussions/144#discussioncomment-6469293>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BA37AE4GHQ6REVBW7BXLF5TXQVMINANCNFSM6AAAAAAZVKVOKM>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

 [https://www.groupeterre.org/sites/groupeterre/files/Logo_pour_signature/logo-mail-AT.png] <https://www.autreterre.org>


Nathalie Talmasse


Assistante de direction



Tél:+32 4 240 68 40


Fax:+32 4 228 07 50


***@***.******@***.***>


Rejoignez Autre Terre sur Facebook<http://www.facebook.com/ongautreterre>



Autre Terre asbl


4e Avenue, 45


B-4040 Herstal


Dossiers et Comptabilité




[https://www.groupeterre.org/sites/groupeterre/files/Logo_pour_signature/iconNewsAT.png]Inscrivez-vous à la newsletter d'Autre terre<https://my.sendinblue.com/users/subscribeembed/js_id/2ml2i/id/1>

**Comment by SiimKirjanen** [comment]
You need to be logged in. Then you will see a form
![request-form](https://github.com/SiimKirjanen/seatreg/assets/21058594/a1c71be2-0fbe-46b8-83cf-0ae25f20ff54)

**Comment by NathalieTalmasse** [comment]
Good Morning,

I’m sorry but I can not find a form following the link, only a forum, and therefore I can not request to be a translation editor.

Could you please tell me the different steps to find the adequate form ?
How long will it take to become a translation editor in French ?
And from there how long will it take for us to approve the translation in French ?

Thanks for the tips,
Nathalie


Nathalie Talmasse





Autre Terre asbl<www.autreterre.org>


Tél:+32 4 240 68 40


N° Int:6840



Fax:+32 4 228 07 50

De : Siim Kirjanen ***@***.***
Envoyé : mardi 18 juillet 2023 12:53
À : SiimKirjanen/seatreg ***@***.***>
Cc : Nathalie Talmasse ***@***.***>; Author ***@***.***>
Objet : Re: [SiimKirjanen/seatreg] Our translation is waiting for validation (Discussion #144)


You need to be logged in. Then you will see a form
[request-form]<https://user-images.githubusercontent.com/21058594/254241107-a1c71be2-0fbe-46b8-83cf-0ae25f20ff54.PNG>

—
Reply to this email directly, view it on GitHub<https://github.com/SiimKirjanen/seatreg/discussions/144#discussioncomment-6477063>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BA37AE3QWCAKAXMF65FCD3TXQZTKRANCNFSM6AAAAAAZVKVOKM>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

 [https://www.groupeterre.org/sites/groupeterre/files/Logo_pour_signature/logo-mail-AT.png] <https://www.autreterre.org>


Nathalie Talmasse


Assistante de direction



Tél:+32 4 240 68 40


Fax:+32 4 228 07 50


***@***.******@***.***>


Rejoignez Autre Terre sur Facebook<http://www.facebook.com/ongautreterre>



Autre Terre asbl


4e Avenue, 45


B-4040 Herstal


Dossiers et Comptabilité




[https://www.groupeterre.org/sites/groupeterre/files/Logo_pour_signature/iconNewsAT.png]Inscrivez-vous à la newsletter d'Autre terre<https://my.sendinblue.com/users/subscribeembed/js_id/2ml2i/id/1>

**Comment by SiimKirjanen** [comment]
I think you used the correct form and I see the request.
![Capture](https://github.com/SiimKirjanen/seatreg/assets/21058594/a0e8015c-8cc1-4813-9295-937b895eee94)
But I am not sure why no one has replied. There is some guide in https://make.wordpress.org/polyglots/handbook/translating/how-to-translate/#i-submitted-a-translation-now-what that may help.
Can't really tell how long it will take. It all depends how fast you get the permissions.

**Comment by NathalieTalmasse** [comment]
Good morning,

There is a long time I didn’t contact you.
In fact we have a problem with our seatreg website once we try to export files in .XLSX format.
We receive this error message :
***@***.***
Meaning : « impossible to open the file « Terre flea-market » because its format or its extension is not valid.  Please check that the file is not damaged ».
And once the file opens it is empty.

Could you help us ?
Thanks in advance,
Nathalie


De : Nathalie Talmasse
Envoyé : vendredi 28 juillet 2023 10:38
À : SiimKirjanen/seatreg ***@***.***>; SiimKirjanen/seatreg ***@***.***>
Cc : Author ***@***.***>; Vincent Oury ***@***.***>; Nicolas Chiapparo ***@***.***>
Objet : RE: [SiimKirjanen/seatreg] Our translation is waiting for validation (Discussion #144)

Good Morning,

I’m sorry but I can not find a form following the link, only a forum, and therefore I can not request to be a translation editor.

Could you please tell me the different steps to find the adequate form ?
How long will it take to become a translation editor in French ?
And from there how long will it take for us to approve the translation in French ?

Thanks for the tips,
Nathalie


Nathalie Talmasse






Autre Terre asbl<www.autreterre.org>


Tél:+32 4 240 68 40


N° Int:6840



Fax:+32 4 228 07 50

De : Siim Kirjanen ***@***.***
Envoyé : mardi 18 juillet 2023 12:53
À : SiimKirjanen/seatreg ***@***.******@***.***>>
Cc : Nathalie Talmasse ***@***.******@***.***>>; Author ***@***.******@***.***>>
Objet : Re: [SiimKirjanen/seatreg] Our translation is waiting for validation (Discussion #144)


You need to be logged in. Then you will see a form
[request-form]<https://user-images.githubusercontent.com/21058594/254241107-a1c71be2-0fbe-46b8-83cf-0ae25f20ff54.PNG>

—
Reply to this email directly, view it on GitHub<https://github.com/SiimKirjanen/seatreg/discussions/144#discussioncomment-6477063>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BA37AE3QWCAKAXMF65FCD3TXQZTKRANCNFSM6AAAAAAZVKVOKM>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

 [https://www.groupeterre.org/sites/groupeterre/files/Logo_pour_signature/logo-mail-AT.png] <https://www.autreterre.org>


Nathalie Talmasse


Assistante de direction


Tél:+32 4 240 68 40



Fax:+32 4 228 07 50


***@***.******@***.***>


Rejoignez Autre Terre sur Facebook<http://www.facebook.com/ongautreterre>



Autre Terre asbl


4e Avenue, 45


B-4040 Herstal


Dossiers et Comptabilité




[https://www.groupeterre.org/sites/groupeterre/files/Logo_pour_signature/iconNewsAT.png]Inscrivez-vous à la newsletter d'Autre terre<https://my.sendinblue.com/users/subscribeembed/js_id/2ml2i/id/1>

**Comment by SiimKirjanen** [comment]
Hello.
Where do you see the error "impossible to open the file"? Are you using some XLSX reader? Do you have any server side error logs? Any info would help as I don't know how to reproduce the issue.

**Comment by NathalieTalmasse** [comment]
Hello,
The message appears after downloading the xls file and trying to open it :
***@***.***

Nathalie


Nathalie Talmasse





Autre Terre asbl<www.autreterre.org>


Tél:+32 4 240 68 40


N° Int:6840



Fax:+32 4 228 07 50

De : Siim Kirjanen ***@***.***
Envoyé : mardi 3 mars 2026 14:35
À : SiimKirjanen/seatreg ***@***.***>
Cc : Nathalie Talmasse ***@***.***>; Author ***@***.***>
Objet : Re: [SiimKirjanen/seatreg] Our translation is waiting for validation (Discussion #144)


Hello.
Where do you see the error "impossible to open the file"? Are you using some XLSX reader? Do you have any server side error logs? Any info would help as I don't know how to reproduce the issue.
